### ISSUE TITLE
Accessibility: Redundant Topics links

### ISSUE THREAD

**Issue by aparkening** [alistapart/AListApart#222]
From Andrew's accessibility audit:

The Topics page has redundant links for each section.  The image and the title for the section should be in a single link.  
![topics-links](https://cloud.githubusercontent.com/assets/7241111/7903734/72236acc-07b3-11e5-8dc1-ddf5688e7812.png)

Current: 
```
<figure>    
<a href="/topics/design"> <img alt="" src="http://alistapart.com/d/topic-illustrations/design.jpg"></img> </a>
 </figure>
 <h1><a href="/topics/design">Design</a></h1>
```

Should be:
```
<a href="/topics/design">
 <figure><img alt="" src="http://alistapart.com/d/topic-illustrations/design.jpg"></img> </figure>   <h2>Design</h2> 
</a>
```

This is a WCAG 2.0 1.1.1 failure because the image has null alternative text within a link where there is no other content.
